conduct/code_of_conduct.md

```diff
@@ -1,131 +1,145 @@
    will depend on the work of others. Any decision you take will affect users
    and colleagues, and you should take those consequences into account when
-   making decisions. Remember that we're a world-wide community. You may be
+   making decisions.    
+   Remember that we're a world-wide community, and that you may be
    communicating with someone with a different primary language or cultural
-   background.
+   background. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners.
+   Do not insult or put down other community members. A community where people
+   feel uncomfortable or threatened is not a productive one.    
+   
+   
+* **3. Consider Impact vs. Intent**.    
+   Disagreements, both social and
+   technical, happen all the time and Pangeo is no exception. People perceive their own behaviors very differently than others perceive them. What motivates one person may cause stress for another. Few steps worth remembering:
+
+   * Other people do not see things the same way you do, do not feel the same as you, or interpret the world the way you do. Consider different people’s perspectives on things like change, drive, structure, process, big picture, and small details.
+   * Ask for feedbacks and listen to them. 
+   * Blaming each other doesn’t get us anywhere, while we can learn from mistakes to find better solutions. 
+
+* **4. Moderate your expectations**.    
+   Please respect that community members choose
+   how they spend their time in the project. A thoughtful question about your
+   expectations is preferable to demands for another person's time.
+
+* **5. A simple apology can go a long way**.    
+   It can often de-escalate a situation,
+   and telling someone that you are sorry is an act of empathy that doesn’t
+   automatically imply an admission of guilt.
 
-4. **Be respectful**. Not all of us will agree all the time, but disagreement is
-   no excuse for poor behavior or poor manners. We might all experience some
-   frustration now and then, but we cannot allow that frustration to turn into a
-   personal attack. It’s important to remember that a community where people
-   feel uncomfortable or threatened is not a productive one.
 
-5. **Be careful in the words that you choose**. Be kind to others. Do not insult
-   or put down other community members. Harassment and other exclusionary
-   behavior are not acceptable. This includes, but is not limited to:
+Rather than considering this code an exhaustive list
+of things that you *can’t* do, take it in the spirit it is intended - a guide to
+make it easier to enrich all of us and the communities in which we participate. 
+By embracing the following principles, guidelines and actions to follow or
+avoid, you will help us make Pangeo a welcoming and productive community. Feel
+free to contact the Code of Conduct Committee at
+[*TODO*](mailto:TODO) with any questions.
```

Comment:
Looks like we need to identify a Code of Conduct Committee and get them an email address.

I'm happy to be included on this committee.  I've looked at my fair share of CoCs over the last few years.

Reply:
Yes this needs to be defined and created.

Reply:
Project Jupyter has a google form for this: https://goo.gl/forms/sJzOIie3zde9M71T2

@rabernat - do you know how we can get a few email addresses using the pangeo.io domain? This would be conduct@pangeo.io